profiles/migrations/0023_remove_newsletter_fields.py

```diff
@@ -0,0 +1,15 @@
+# Generated by Django 2.2.10 on 2020-03-30 06:22
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("profiles", "0022_add_newsletter_options"),
+    ]
+
+    operations = [
+        migrations.RemoveField(model_name="profile", name="newsletters_via_email",),
+        migrations.RemoveField(model_name="profile", name="newsletters_via_sms",),
+    ]
```

Comment:
Yep. Kept them because there were other contributors to the feature branch. They will cancel out when migrations are eventually squashed at some point.